doc/man1/openssl-req.pod.in

```diff
@@ -90,8 +90,8 @@ The data is a PKCS#10 object.
 =item B<-cipher> I<name>
 
 Specify the cipher to be used for encrypting the private key.
-The default cipher is 3DES (DES-EDE3-CBC).
-If no cipher is specified, 3DES will be used by default.
+The default cipher is AES-256-CBC.
```

Comment:
You can remove this sentence as it just duplicates the information in the second sentence.